app/controllers/admin/people_controller.rb

```diff
@@ -3,5 +3,32 @@ class PeopleController < AdminController
     def index
       @people = Person.all
     end
+
+    def batch_update
+      changes_made = false
+      people_params.each do |param|
+        person = Person.find(param[:id])
+        next if person.admin? == param[:admin]
+        person.toggle_admin!
+        changes_made = true
+      end
+      if changes_made
```

Comment:
I don't know that we care that changes are made or not. I would only care if the `save` failed